fifteen_min/index.html

```diff
@@ -148,24 +132,10 @@ <h2>Loading...</h2>
             <p>(Your browser must support WebGL 2.0 and WebAssembly)</p>
```

Comment:
Can you remove the "2.0" here and in all the others please?

Reply:
Good catch - fixed!